custom_components/alexa_media/config_flow.py

```
@@ -1072,16 +1098,52 @@ async def wrapped(request, **kwargs):
                 cls.known_ips[request.remote] = datetime.datetime.now()
+            _LOGGER.warning(
+                "PROXY DEBUG >>> Request: %s %s | Remote: %s | Headers: %s",
+                request.method,
+                request.url,
+                request.remote,
+                dict(request.headers),
+            )
             try:
-                return await cls.handler(request, **kwargs)
-            except httpx.ConnectError as ex:  # pylyint: disable=broad-except
+                result = await cls.handler(request, **kwargs)
```

Pylint (4.0.4)

[error] cls.handler is not callable

(E1102)